src/WrapperApp/components/Results/ResultsPanel.tsx

```diff
@@ -28,19 +28,21 @@ function ResultsPanel() {
 	const [estimatorsResults, setEstimatorsResults] = useState<EstimatorResults[]>([]);
 	const [groupQuantities, setGroupQuantities] = useState(false);
 
-	const userTabInputFilesEstimatorNames = simulation?.input.userInputFilesEstimatorNames?.map(
-		output => output.slice(0, -1)
+	const userTabInputFilesEstimatorNames = simulation?.input.estimatorsItems?.map(output =>
+		output.name.slice(0, -1)
 	);
 	const uploadedInputFilesEstimatorNames = estimatorsResults?.map(estimator => estimator.name);
 
 	const editorEstimatorNames = simulation?.input.inputJson?.scoringManager.outputs
 		.filter(output => output.name)
 		.map(output => output.name);
 
-	const estimatorsTabData: string[] | undefined = simulation?.input.userInputFilesEstimatorNames
+	const estimatorsTabData: string[] | undefined = simulation?.input.estimatorsItems
 		? userTabInputFilesEstimatorNames
 		: editorEstimatorNames;
 
+	const estimatorsPagesData = simulation?.input.estimatorsItems;
```

Comment:
can you describe in the comment what is the purpose of estimatorItems ? what will be kept there ?